### DISCUSSION TITLE
tapping mechanism on touch device

### DISCUSSION THREAD

**Discussion by DoriteLover** [ppy/osu#30120] [Ideas] [answered]
when I tried lazer (moving from droid) the tapping mechanism is different. when there is a stream to slider such as : 1 2 3 4 5 6----- . and when I don't hold on the 6 the slider end will not count which is kind of annoying for me but for the osu droid unofficial version when I click mid slider the holding will not release but for lazer it will release which is not gonna have a slider end and combo gone which is very annoying 
![Screenshot_20241005_143347_Samsung Notes](https://github.com/user-attachments/assets/f05edd66-2b76-4c2b-a225-744abb48e547)
this is comparing tapping mechanism

**Comment by DoriteLover** [comment]
this is osu droid
https://github.com/user-attachments/assets/284688ca-f5e6-4b22-a1c8-6f5927e5ff84

this is lazer, I forgot to show taps so pay attention to key overlay
https://github.com/user-attachments/assets/3f456122-b2a5-49ea-bab0-1a1fc03bea81

**Reply by peppy** [reply, marked as answer]
You're not holding a key? You can't expect this to work.

As in you're pressing another button but not keeping it pressed. You should either not press it or keep it pressed. We can't provide what you want because it would be taking difficulty away from the game (basically giving you three valid pressable keys).

**Reply by DoriteLover** [reply]
fair enough
